app/views/event_categories/show.html.erb

```diff
@@ -1,6 +1,7 @@
 <% @search_component = Events::SearchComponent.new(
         @event_search,
         @form_action.to_s,
+        @performed_search,
         include_type: false,
         heading: "Search for #{@category_name}",
         allow_blank_month: true,
```

Comment:
Rather than using an instance variable here I feel like it should be a `content_for :search_component` block, with a corresponding `yield :search_component` in the layout.